kratos/utilities/data_type_traits.h

```diff
@@ -212,6 +270,30 @@ template<class TDataType> class DataTypeTraits
         rValue = *pContiguousDataBegin;
     }
 
+    /**
+     * @brief Copies contiguous values to the container.
+     *
+     * This method copies all the contiguous values in the given @ref pContiguousDataBegin pointer
+     * to the @ref rContainer.
```

Comment:
same `doxygen` stuff; check all your annotations plz

```suggestion
     * to the @p rContainer.
```

Reply:
fixed all